src/main/java/com/elastic/support/rest/RestClientBuilder.java

```diff
@@ -259,5 +266,18 @@ public RestClient build() {
 
     }
 
+    /**
+     * This overrides any hostname mismatch in the certificate
+     */
+    private class BypassHostnameVerifier implements HostnameVerifier {
```

Comment:
Does this class actually get used?

Reply:
No, it was a test to see why the NoVerify was not working. It did actually used to be in there and was recently removed because Apache provided a built in class. Just left it in because I was in a hurry and didn't notice it until after I'd pushed it up.